### PR TITLE
Upgrade to Safari Technology Preview 94 on macOS Mojave

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
infrastructure/ expectations need to be updated, but in the meantime https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=34727 is a full run in progress.

File: infrastructure/metadata/infrastructure/testdriver/bless.html.ini [hunk 1]
```diff
@@ -0,0 +1,31 @@
+[bless.html]
```
Comment (foolip) [inline infrastructure/metadata/infrastructure/testdriver/bless.html.ini]: @LukeZielinski as an FYI, this expectation file is probably the best example so far of something that's time-consuming to write manually. I had to copy the section the right number of times and copy all the names from log output until I got it right...

**Review by jgraham — APPROVED** [review]

File: infrastructure/metadata/infrastructure/testdriver/bless.html.ini (hunk 1 above)
Comment (foolip) [reply infrastructure/metadata/infrastructure/testdriver/bless.html.ini]: Context: https://github.com/web-platform-tests/wpt/issues/12697, and it's easier to update expectations to match actual results in Chromium, or at least I know how to do it there.

**Comment by stephenmcgruer** [comment]
In terms of the regressions from 82 to 94, some of them seem to be MISSING results. I noticed some interpolation tests had regressed which was surprising, but on inspection they're just missing: https://wpt.fyi/results/css/css-grid?q=seq%28status%3Aok%7Cstatus%3Apass%20status%3A%21ok%26status%3A%21pass%29&run_id=313540021&run_id=342290025

Not sure what this means, can you take a look @foolip?

**Comment by foolip** [comment]
The way to start debugging that is to download the logs from https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=34727 and grep for grid-template-columns-interpolation.html. Looks like it appears in "all tests: Safari Technology Preview 3" and that job seems to end as expected.

Turns out this is just an artifact of applying the `seq(status:ok|status:pass status:!ok&status:!pass)` filter, because that excludes all the subtests that don't match the filter. If you remove that, you'll see that actually a bunch of subtest names have changed, overall passrate is unchanged.

Do the subtest names depend on the implementation here? If so, that should be fixed :)

**Comment by stephenmcgruer** [comment]
> Do the subtest names depend on the implementation here? If so, that should be fixed :)

Turns out they do, sent https://github.com/web-platform-tests/wpt/pull/19759

**Pull request merged**